lms/templates/courseware/progress.html

```diff
@@ -80,6 +80,13 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
+                  %elif not is_id_verified or is_unverified:
+                   <div class="msg-content">
+                    ## Translators: This message appears to users when the users have not completed identity verification.
+                    <h2 class="title">${_("Certificate unavailable")}</h2>
+                    <p class="copy">${_("Your certificate is not issued because you do not have a current verified identity with edX. ")} <a href="${reverse('verify_student_reverify')}"> ${_("Verify your identity now.")}</a></p>
```

Comment:
Got it. 
BTW, Can we remove the wording `with edX`? I think it's a better experience for openedx community as well as it makes the message a bit shorter. As it stands now, it would stretch the message to second line with just 1 word. See screenshot:

![second_line](https://cloud.githubusercontent.com/assets/16839373/15399400/7760e0c8-1db5-11e6-8148-8ce81162a484.jpeg)